packages/playwright/src/reporters/blob.ts

```diff
@@ -58,7 +58,7 @@ export class BlobReporter extends TeleReporterEmitter {
     const metadata: BlobReportMetadata = {
       version: currentBlobReportVersion,
       userAgent: getUserAgent(),
-      name: process.env.PWTEST_BLOB_REPORT_NAME,
+      name: config.botName || process.env.PWTEST_BLOB_REPORT_NAME,
```

Comment:
We could rename `name` to `botName` in the blob format too, but that would require migration from v1 -> v2, so I'm hesitant for now.